pkg/test/framework/components/echo/config.go

```diff
@@ -85,6 +85,9 @@ type Config struct {
 
 	// The set of environment variables to set for `DeployAsVM` instances.
 	VMEnvironment map[string]string
+
+	// Used to supply a non default echo image for some testing and debugging situations
+	OverrideK8SImage string
```

Comment:
do you want this to be a flag passed to `go test`? or are you writing a custom test which needs a custom image?